pkg/features/versioned_kube_features.go

```diff
@@ -404,6 +404,7 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 
 	ImageVolume: {
 		{Version: version.MustParse("1.31"), Default: false, PreRelease: featuregate.Alpha},
+		{Version: version.MustParse("1.33"), Default: false, PreRelease: featuregate.Beta},
```

Comment:
I see the validation change happening, so it makes sense to wait to default-enable until the next release.

I haven't followed the feature, what's the purpose behind promoting to beta but keeping disabled by default?

Reply:
That decision was made as part of the containerd support discussion: https://github.com/haircommander/enhancements-1/blob/edb4738bb0f5b7ee8a6021c51263ad0bde099fc8/keps/sig-node/4639-oci-volume-source/README.md#beta

Now, containerd has merged https://github.com/containerd/containerd/pull/10579 which is not part of any release yet. So I assume this still holds valid.